error/src/lib.rs

```diff
@@ -252,6 +252,10 @@ impl CaliptraError {
 
     /// FMC Errors
     pub const FMC_RT_ALIAS_UNIMPLEMENTED: CaliptraError = CaliptraError::new_const(0x000F0001);
+    pub const FMC_HANDOFF_INVALID_PARAM: CaliptraError = CaliptraError::new_const(0x000F0002);
```

Comment:
going forward, I would like to see FMC_HANDOFF_INVALID_PARAM split into more specific error codes. If I am reading this right, we have 16 bits worth of codes just from FMC right?